### PR TITLE
First Code Review

### PR DESCRIPTION
**Pull request by David405**

### CODE REVIEW

**Comment by Ispirett** [comment]
# Reviewing

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey, @David405  @dev1980  really great job so so far :1st_place_medal: You guys did a great  job with the layout. There are just a few  things to take care of  :+1:  

- [ ] The responsiveness of the site needs some adjusting.
### Breakpoints.
small: max-width around 700
medium: max-width around 1000
large: max-width over 1000

**you have implemented two of them** 
You are using `min-width` which is not an issue if you use it properly. 
But the main issue is the site is not fully responsive. 
For example: on line `1012` in your CSS file , you have only setup  adjustments for `nav-search` and `time`
 
**I encourage you to watch the original site and try to mimic the changes you see when you resize the browser.  You can always refer to information from the Odin project on responsive design.**

- [ ] HTML errors
Also,  guys, There are a lot of HTML errors on this page use this tool and please correct them https://validator.w3.org/


**Also, remember You should use Github/Git Flow in this project. However, changing it now would take too much time.
Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description.**

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Nice work team :clap: 
Your project looks really nice and you should be proud of it.

But there is still one thing that we need to change. I notice that the stickler is not verifying your project. Maybe this is happening because it's not set up properly. Please check the instruction on the pathwrite on how to set up the linter and make it work on this project. You will also need to fix errors(if the linter found any). I know that in the past we didn't pay much attention to it but linting is important to reduce errors and improve the overall quality of your code. Using the linter also gives a professional look in your repository. Please kindly make this change and submit for code review again.

[FOR FUTURE PROJECT]
 You should use Github/Git Flow in future projects. It's really easy to use it and will make your repository looks more professional. You will find instructions in the Patwright lesson description.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Nice work team :clap: 
Your project looks really nice and you should be proud of it.

But there is still one thing that we need to change. I notice that the stickler is not verifying your project. Maybe this is happening because it's not set up properly. Please check the instruction on the pathwrite on how to set up the linter and make it work on this project. You will also need to fix errors(if the linter found any). I know that in the past we didn't pay much attention to it but linting is important to reduce errors and improve the overall quality of your code. Using the linter also gives a professional look in your repository. Please kindly make this change and submit for code review again.

[FOR FUTURE PROJECT]
 You should use Github/Git Flow in the future

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey @dev1980 @David405 I can see the stickler problem still exists. 

You can try use this in your stickler file instead  :+1: https://github.com/Ispirett/newsweek/blob/master/stickler.yml

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Nice Job Team

You almost there :smile: 

Since the main objective of the project is to make a **responsive** clone of [this page](https://thenextweb.com/) I would suggest making sure [this section]() (shown below) is responsive and have the small fixed navbar on smaller screens.

| Clone | Source Page |
| :-----: | :-----: |
| tnw clone reponsive  | tnw source reponsive  |

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Comment by Oghenebrume50** [comment]
Reviewing......

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi guys good job so far kudos on your progress, I still have to stress on the responsiveness, it is still not responsive, please compare with the original site display at each breakpoint to know what to do. 


From the screenshot above you can see it is not displayed properly. And avoid leaving spaces around containers where it is not needed. 

Please fix that and submit for re-review

File: css/.stickler.yml
Comment (bolah2009) [inline css/.stickler.yml:5]: `HINT`

Use `.stickler.yml` file in the root path of the project.

File: eslint.config
Comment (bolah2009) [inline eslint.config:9]: `HINT`

This config file is not relevant to this project. Please, delete. :+1:

File: .stickler.yml
Comment (bolah2009) [inline .stickler.yml:10]: `HINT`

Yes, this is the right location for this file. :smile: But with the wrong content. :worried: 

Let's use [this file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml) instead.

``` yml
linters:
  stylelint:
    config: 'stylelint.config.js'
```
- Also, add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).

**Review by bolah2009 — APPROVED** [review]
## Good Progress. :tada: 

The [clone](https://raw.githack.com/dev1980/BUILDING-WITH-RESPONSIVE-DESIGN/development/index.html) looks good to me now and responsive. :+1: 

### It's a :+1: from me.

### Although you can merge now, I strongly recommend you check the comments below and add the appropriate `linter files`.

- [Next Project] Use GitHub Flow: For HTML & CSS projects, use only one feature branch (e.g. `embedding-images-and-video`, `html-forms`, `responsive-design`, etc) and **do not merge any feature branch to master until it's been approved**.


Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### Note that comments tagged `OPTIONAL` are strongly recommended but not required to approve the project.

**Comment by dev1980** [comment]
Final code review approved

**Pull request merged**
